debian/README.debian.md

```diff
@@ -0,0 +1,27 @@
+# Build image
+
+To build the image, run:
+```
+make debian
+```
+
+# Run the container
+
+Run the machine with:
+```
+docker run -d linuxbrew/debian tail -f /dev/null
```

Comment:
I used this Stackoverflow answer : https://stackoverflow.com/a/30209974/1807667
but `docker run -it` is better for this case. I will correct it.